xml/soc-operations-maintenance.xml

```diff
@@ -31,8 +31,11 @@
   <note>
    <title>No Maintenance Mode</title>
```

Comment:
This title should be changed since there *is* a maintenance mode, but at the Pacemaker level rather than in OpenStack.